quantum/rgb_matrix.c

```diff
@@ -41,6 +41,10 @@ rgb_config_t rgb_matrix_config;
     #define EECONFIG_RGB_MATRIX EECONFIG_RGBLIGHT
 #endif
 
+#ifndef RGB_MATRIX_MAXIMUM_BRIGHTNESS
```

Comment:
It may be a good to limit the value too, so that we can't have a value higher than 255. 

so maybe instead, something like: 
```c
#if !defined(RGB_MATRIX_MAXIMUM_BRIGHTNESS) || RGB_MATRIX_MAXIMUM_BRIGHTNESS > 255
    #define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255
#endif
```

Reply:
cool yea agreed!